Update existing content in key differences page

pedrohyvo opened the pull request:
Update existing content in key differences page. This PR is a part of #3 .

pedrohyvo commented:
@wlsf82 @lucastfa-toptal can one of you guys review this PR, please?

wlsf82 commented:
@pedrohyvo, is this just changing the line length (no content changes)? If so, it looks good to me.

pedrohyvo commented:
> @pedrohyvo, is this just changing the line length (no content changes)? If so, it looks good to me.

@wlsf82 I just added some links and fixed few typos. I will merge it since they are small changes
